.vscode/launch.json

```diff
@@ -16,7 +16,7 @@
 			"outFiles": [
 				"${workspaceFolder}/dist/**/*.js"
 			],
-			"preLaunchTask": "npm: compile"
+			"preLaunchTask": "adoc-prelaunch"
```

Comment:
It's likely not well documented but this is under the package.json script: `vscode: prepublish`

I don't think we should have two tasks with the same function so I think we could remove this.